how_to/create_a_local_bpd_chapter.md

```diff
@@ -1 +1,12 @@
 # How to Create a Local BPD Chapter
+
+To create one in your local community, send in your email request to meetups@blackpythondevs.com
```

Comment:
Well done Rahmat! Can we slightly improve this:

To create a BPD chapter in your local community, send an email request to `meetups@blackpythondevs.com`.